Gems/Atom/Tools/AtomToolsFramework/Code/Include/AtomToolsFramework/Graph/GraphCompilerRequestBus.h

```diff
@@ -24,7 +23,10 @@ namespace MaterialCanvas
         // Get a list of all of the generated files from the last time this graph was compiled.
         virtual const AZStd::vector<AZStd::string>& GetGeneratedFilePaths() const = 0;
 
-        // Evaluate the graph nodes, slots, values, and settings to generate and export shaders, material types, and materials.
+        // Get the graph export path based on the document path or default export path.
+        virtual AZStd::string GetGraphPath() const = 0;
```

Comment:
Maybe not in this PR, but would it be worthwhile to have this return an `AZ::IO::Path` instead of a string?